chinglee-iot/FreeRTOS
# UT: Framework to add mocking to unit testing smp

SMP config assert 

Description
-----------
Start a  framework with 1 test for SMP config assert testing coverage

Test Steps
-----------
configASSERT is hit and code is covered


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
